maps/away/magshield/magshield.dm

```diff
@@ -0,0 +1,136 @@
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "rad_sensor"
+	anchored = TRUE
+
+/obj/structure/magshield/nav_light
+	name = "navigation light"
+	desc = "Large and bright light regularly emitting green flashes."
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "nav_light_green"
+	anchored = TRUE
+	density = TRUE
+
+/obj/structure/magshield/nav_light/New() //Would be cool if these could flash, but I couldn't figure out how to carry that over.
+	..()
+	set_light(1.2, 3, LIGHT_COLOR_GREEN)
+
+
+/obj/structure/magshield/nav_light/red
+	desc = "Large and bright light regularly emitting red flashes."
+	light_color = "#ee0000"
+	icon_state = "nav_light_red"
+
+/obj/structure/magshield/nav_light/red/New()
+	..()
+	set_light(1.2, 3, LIGHT_COLOR_RED)
+
+/obj/item/book/manual/magshield_manual
+	name = "SOP for Planetary Shield Orbital Station"
+	icon = 'maps/away/magshield/magshield_sprites.dmi'
+	icon_state = "mg_guide"
+	author = "Unknown"
+	title = "Standard operating procedures for Planetary Shield Orbital Station"
+
+	dat = {"<html>
+				<head>
+				<style>
+				h1 {font-size: 18px; margin: 15px 0px 5px;}
+				h2 {font-size: 15px; margin: 15px 0px 5px;}
+				h3 {font-size: 13px; margin: 15px 0px 5px;}
+				li {margin: 2px 0px 2px 15px;}
+				ul {margin: 5px; padding: 0px;}
+				ol {margin: 5px; padding: 0px 15px;}
+				body {font-size: 13px; font-family: Verdana;}
+				</style>
+				</head>
+				<body>
+
+				<h1>Introduction</h1>
+				...is happy to see you as our customer! Please read this guide before using and operating with your custom PSOS - Planetary Shield Orbital Statiion.
+				<h2>Best uses for PSOS</h2>
+				PSOS is intended for protecting exoplanets from high energy space radiation rays and particles in the interest of furthering research on this phenomena. Best used for planets lacking active geomagnetic field so PSOS would compensate its absence and provide more data.<br>
+				<h2> Applied technologies</h2>
+				...is delivering you your new PSOS with set of four (4) high-strength magnetic field generators. Those devices use rotating supeconducter hands to create magnetic field with strength up to 5 Tesla effectively deflecting up to 99% of space radiation spectrum.<br>
+				<br>
+				Special modified vacuum radiation sensors will help you evaluate radiation level and adjust power input of PSOS magnetic generators for best efficiency and power saving.
+				<br><br><br>
+				<i>rest of the book pages are gone</i>
+				</body>
+			</html>
+			"}
+
+/obj/item/paper/magshield/tornpage
+	name = "torn book page"
+	info = "...you must carefully control radiation sensor automatics during solar flares. Sudden burst of high-energy plasma may cause positive feedback loop and increase magnetic genretors output in order of magnitude. This situation would lead to general damage of unprotected electronic devices as well as trajectory changes in nearby nickel-ferrum astero#&$"
```

Comment:
```suggestion
	info = "...you must carefully control radiation sensor automatics during solar flares. Sudden burst of high-energy plasma may cause positive feedback loop and increase magnetic genretors output in order of magnitude. This situation would lead to general damage of unprotected electronic devices as well as trajectory changes in nearby nickel-ferrum asteroids."
```